cobalt/build/configs/BUILD.gn

```diff
@@ -32,3 +37,22 @@ config("default_includes") {
     ]
   }
 }
+
+config("buildflag_defines") {
+  defines = []
+  if (is_cobalt) {
+    defines += [ "ENABLE_BUILDFLAG_IS_COBALT" ]
+  }
+  if (is_starboard) {
+    defines += [ "OS_STARBOARD" ]
```

Comment:
Can we get rid of `OS_STARBOARD` and just have a `use_starboard` variable that defines `BUILDFLAG_INTERNAL_USE_STARBOARD`?

Reply:
We can, my idea here was to mimic the other platform macros in //build/build_config.h (i.e. OS_CHROMEOS). But of course that makes it inconsistent with the rest of the defines here, which looks pretty strange. I'm fine with changing it, but maybe take a look a the other OS_FOO defines in build_config.h and see if you still would prefer to?

Reply:
See https://github.com/youtube/cobalt/blob/232171bbaefb3afb59b1a47abde13831d38e2612/build/config/linux/BUILD.gn#L40 and https://github.com/youtube/cobalt/blob/232171bbaefb3afb59b1a47abde13831d38e2612/build/build_config.h#L79

Reply:
Ah yeah I see. I'm ok with keeping it.